sig-security/security-audit-2021/RFP.md

```diff
@@ -136,3 +136,36 @@ Separately, any code released with or as part of the report needs to be under th
+
+### Do you have any developer documentation or design documentation specifications that aren't available on the internet that you would be able to share?
+
+Kubernetes is an open source project, all documentation is available on https://kubernetes.io or on https://github.com/kubernetes.
+
+### What are the most important publicly available pages detailing the design of the system and the data it receives.
+
+- Overview of [Kubernetes components](https://kubernetes.io/docs/concepts/overview/components/)
+- [kube-apiserver overview](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/)
+- [kube-scheduler overview](https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/)
+- [Operating etcd clusters for Kubernetes](https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/)
+- [etcd clustering guide](https://etcd.io/docs/next/op-guide/clustering/)
+- [kube-controller-manager overview](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/)
+- [cloud-controller-manager overview](https://kubernetes.io/docs/concepts/architecture/cloud-controller/)
+- [cloud-controller-manager administration](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/)
+- [kubelet overview](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)
+- [kube-proxy overview](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/)
+- [secrets-store-csi-driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver)
+
+### How long does the Working Group envision the engagement lasting and what is the latest date you can receive the deliverables?
+
+The latest date to receive deliverables will be negotiated with the selected vendor.
+
+### Which attack vectors are of most concern to the Working Group.
+
+1. The attack vector most concerned about is unauthenticated access to a cluster resulting in compromise of the [components in-scope](#project_goals_and_scope)
+2. Crossing cluster boundaries for multi-cluster configuration
+3. Crossing namespaces
+4. Any attack vector that exists against the components in scope
```

Comment:
```suggestion
4. Any other attack vector that exists against the components in scope
```

Reply:
@tabbysable updated the PR with the suggested change 
> 4. Any other attack vector that exists against the components in scope